sdk/core/core-http/lib/webResource.ts

```diff
@@ -74,6 +74,10 @@ export class WebResource {
 
   /** Callback which fires upon download progress. */
   onDownloadProgress?: (progress: TransferProgressEvent) => void;
+  /**
```

Comment:
nit: blank line before start of comment block

Reply:
Fixed, thanks!